responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -63,20 +155,21 @@ def __init__(self,
                  classes: Optional[np.ndarray] = None,
                  serializer: Optional[Any] = None,
                  maximum_rows_for_test: int = 5000,
-                 feature_metadata: Optional[FeatureMetadata] = None):
+                 feature_metadata: Optional[FeatureMetadata] = None,
+                 **kwargs):
```

Comment:
it's better to usually list out the parameters if possible... I think with a separate class we won't need this kwargs, is that correct?

Reply:
No. This is not at all about parameters that are for any functionality in RAIInsights, but only for the feature flag. I do not want to show it to people. If they know about it they can use it at their own risk, but otherwise they shouldn't assume that forecasting is already supported. The same mechanism can work for other feature flags in the future.

I agree on not hiding kwargs that we want people to use.